docs/eck-attributes.asciidoc

```diff
@@ -1,5 +1,5 @@
-:eck_version: 2.5.0
+:eck_version: v2.6.0
```

Comment:
```suggestion
:eck_version: 2.6.0
```

What if we kept without the `v` and updated the crds/operator manifest urls and the name of the image in the doc, so we have nothing to do for the last 3 bullets above?